Socket.h

```diff
@@ -58,4 +68,45 @@ class Socket
              << std::endl;
       }
    };
+
+private:
+    
+    void process_agent_input(std::string agent_request) 
+    {
+       switch (agent_request[0])
+       {
+            // Fire left flipper and turn off right flipper
```

Comment:
This is fine for now, but I want to make sure that turning off a flipper doesn't instantly reset it to the off position. It should ideally act like it would in real life, where it would take a little bit of time to go back from the on to the off position. 
Maybe this code already does that, and if so, good job.